src/errorpage.cc

```diff
@@ -218,8 +229,12 @@ class ErrorPageFile: public TemplateFile
 
 protected:
     virtual void setDefault() override {
-        template_ = "Internal Error: Missing Template ";
-        template_.append(templateName.termedBuf());
+        if (const auto defaultTemplate = FindDefaultTemplate(templateCode)) {
+            template_ = defaultTemplate;
```

Comment:
Since setDefault() is called at the end of loadDefault(), we will always get a WARNING message for missing MGR_INDEX:
```
2022/10/29 23:05:36.988| 1,3| errorpage.cc(395) loadDefault: WARNING: failed to find or read error text file MGR_INDEX
```
I decided that we should not add a lot of ifs in this PR to suppress all such WARNINGs - probably we should do it separately. For example, there is already a flag (TemplateFile::silent) for this, but it is not carefully checked before all debugging lines.

Reply:
Hm... Let's forget about the official setDefault() method _name_ for a second. Should not we check and apply FindDefaultTemplate() earlier in TemplateFile::loadDefault(), before this poorly named method is called? AFAICT, the overall loadDefault() method logic allows us to apply default templates before warning about the missing ones.

Reply:
> Should not we check and apply FindDefaultTemplate() earlier in TemplateFile::loadDefault(),

That was what I initially done (9c058eb) but then observed that there are two default error categories: the old one (setDefault() for required templates) and our new one (defaults for optional templates). I'll move it back, but we should probably rename the old setDefault() to avoid confusion.